syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";

message GithubActions {
  message FetchActionRunInfo {
    google.protobuf.StringValue owner = 1;
    google.protobuf.StringValue repo = 2;
    google.protobuf.StringValue workflow_id = 3;
  }

  enum TaskType {
    UNKNOWN = 0;
    FETCH_ACTION_RUN_INFO = 1;
  }

  TaskType type = 1;
  oneof task {
    FetchActionRunInfo fetch_action_run_info = 2;
  }
}